docker/docker-compose.yml

```diff
@@ -4,31 +4,62 @@
 
 version: '3'
 services:
-    db:
+    postgres:
         image: postgres:11.12-alpine
         environment:
             POSTGRES_PASSWORD: password
             POSTGRES_DB: flagsmith
         container_name: flagsmith_postgres
 
-    api:
+    flagsmith:
         image: flagsmith/flagsmith:latest
         environment:
+            # All environments variables are available here:
+            # API: https://docs.flagsmith.com/deployment/locally-api#environment-variables
+            # UI: https://docs.flagsmith.com/deployment/locally-frontend#environment-variables
+        
+        
             DJANGO_ALLOWED_HOSTS: '*' # Change this in production
             DATABASE_URL: postgresql://postgres:password@db:5432/flagsmith
             DISABLE_INFLUXDB_FEATURES: true # set to false to enable InfluxDB 
-            EMAIL_BACKEND: django.core.mail.backends.smtp.EmailBackend
+
+            ENV: prod # set to "prod" in production.
+
             # For more info on configuring InfluxDB - https://docs.flagsmith.com/deployment/overview#influxdb
             # INFLUXDB_URL: http://influxdb:8086
             # INFLUXDB_BUCKET: flagsmith_api
             # INFLUXDB_ORG: # Add your influx org id here
             # INFLUXDB_TOKEN: # Add your influx token here
+
+            # For more info on configuring E-Mails - https://docs.flagsmith.com/deployment/locally-api#environment-variables
+            #
+            # Example SMTP:
+            # EMAIL_BACKEND: django.core.mail.backends.smtp.EmailBackend
+            # EMAIL_HOST: mail.example.com
+            # SENDER_EMAIL: flagsmith@example.com
+            # EMAIL_HOST_USER: flagsmith@example.com
+            # EMAIL_HOST_PASSWORD: smtp_account_password
+            # EMAIL_PORT: 587 # optional
+            # EMAIL_USE_TLS: True # optional
+            
+            # ENABLE_ADMIN_ACCESS_USER_PASS: True # set to True to enable access to the /admin/ Django backend via your username and password
+            # ALLOW_REGISTRATION_WITHOUT_INVITE: True # Determines whether users can register without an invite. Defaults to True. Set to False or 0 to disable. Note that if disabled, new users must be invited via email.
+            
+            
+            # For more info on configuring flagsmith on flagsmith - https://docs.flagsmith.com/deployment/overview#running-flagsmith-on-flagsmith
```

Comment:
We are looking to deprecate running flagsmith on flagsmith as it has been a source of a lot of confusion - this is being worked on in #394 

As a result I think we should remove these helper comments

Reply:
Done :-)